charts/prometheus-operator/Chart.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v2
+description: The chart deployes the Promethes Operator
+icon: https://raw.githubusercontent.com/prometheus/prometheus.github.io/master/assets/prometheus_logo-cb55bb5c346.png
+engine: gotpl
+type: application
+maintainers:
+  - name: langecode
+    email: tal@netic.dk
+  - name: bismarck
+name: prometheus-operator
+sources:
+  - https://github.com/prometheus-community/helm-charts
+  - https://github.com/prometheus-operator/prometheus-operator
+version: 1.0.0
```

Comment:
Imho we should start with 0.1.0

Reply:
Yeah, sure we can do `0.1.0` for a start